source/utilities.tex

```diff
@@ -13606,6 +13606,11 @@
   template<class R, class... ArgTypes>
     bool operator==(const function<R(ArgTypes...)>&, nullptr_t) noexcept;
 
+  // \ref{func.wrap.move}, move only wrapper
+  template<class... S> class move_only_function;        // \notdef
```

Comment:
> Is that actually the intention? 
Yes (someone requested it in one of the LWG reviews, IIRC).  

> What would it mean, that there are overloads of `operator()`?
Presumably so?

Reply:
It does not matter in terms of specification. If an implementation wants, it can even declare the main template as `template <class T, class U = T> class move_only_function`. There is no guarantee that the standard library template has no extra type parameters.
I added a PR at https://github.com/cplusplus/draft/pull/5198

Reply:
It seems that [conforming] doesn't permit that. Or prohibits it by omission.